[OSDOCS-11260] "Enable FIPS cryptography" option (#79375)
openshift/openshift-docs: modules/osd-create-cluster-red-hat-account.adoc

====
+
endif::osd-on-gcp[]
.. Leave *Enable user workload monitoring* selected to monitor your own projects in isolation from Red Hat Site Reliability Engineer (SRE) platform metrics. This option is enabled by default.
ifdef::osd-on-gcp[]
. Optional: Expand *Advanced Encryption* to make changes to encryption settings.
+
.. Optional: Select *Enable FIPS cryptography* if you require your cluster to be FIPS validated.
+
[NOTE]
====

> If  *Enable FIPS cryptography* is selected, *Enable additional etcd encryption* is enabled by default and cannot be disabled. Users can choose to select *Enable additional etcd encryption* without selecting *Enable FIPS cryptography*.

Reviewer: ```suggestion
If *Enable FIPS cryptography* is selected, *Enable additional etcd encryption* is enabled by default and cannot be disabled. You can select *Enable additional etcd encryption* without selecting *Enable FIPS cryptography*.
```
delete extra space
